oak_functions/loader/testdata/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/oak-ci/oak-functions@sha256:98fe36d3538ebff72d809ef6f5d1c38e9b8835569de45c1d1540ba65486a6c0f
```

Comment:
I think it is better to have a separate `oak_functions/examples` folder with examples in it. `testdata` should be only for testing really. But we can fix this later. Perhaps add a TODO somewhere?

Reply:
I hope as soon as we have another example (perhaps the weather one) we can move things around and it will make more sense. Basically we can have a docker image for the example and remove this one.